lib/matplotlib/cbook.py

```diff
@@ -1941,8 +1941,8 @@ def _bootstrap_median(data, N=5000):
         M = len(data)
         percentiles = [2.5, 97.5]
 
-        ii = np.random.randint(M, size=(N, M))
-        bsData = x[ii]
+        bs_index = np.random.randint(M, size=(N, M))
```

Comment:
Much better than before! Using underscored `bs_index` next to camelcased `bsData` is a bit strange... `bsIndices` would be my choice. but I'm nitpicking :)